src/fantasy_forge/shell.py

```diff
@@ -165,6 +166,16 @@ def do_inventory(self, arg: str):
         """shows the contents of the players inventory"""
         print(self.player.inventory.on_look())
 
+    def do_armour(self, arg: str):
+        """shows the players armour"""
+        for armour_type, armour_item in self.player.armour_slots.items():
+            if armour_item is None:
+                print(f"{armour_type}: None")
```

Comment:
Can an armor type be None? If it can we should add a translated message for it, if not, it should be a warning through a logger. If it can't be none we could add a check #16 for it.